docs/platform/integration/bigcommerce/getting-started.md

```diff
@@ -10,14 +10,21 @@ Before you get started you'll need a BigCommerce store and have access to the ad
 
 
 ## 1. Install our app
-<img
-  src="/img/docs/platform/bigcommerce/app.jpg"
-  alt="BigCommerce app"
-  width="200"
-  style={{ marginBottom: "20px", display: "block" }}
-/>
-
-The first thing you'll need to do is visit the [BigCommerce App Store](https://www.bigcommerce.com/apps/) and download our app.
+<a
+  href="https://www.bigcommerce.com/apps/deity-falcon-pwa"
+  target="_blank"
+  rel="noopener noreferrer"
```

Comment:
```suggestion
  rel="noopener"
```
We could leave the referrer in for analytics purposes

Reply:
@nimobeeren - i think I need noreferrer for IE11 https://caniuse.com/#search=noopener - https://caniuse.com/#search=noreferrer